src/util/descriptionParser.tsx

```diff
@@ -1,49 +1,51 @@
 import { createElement, ReactNode } from "react";
 
 const RAW_DESCRIPTION_TAG_NAME_MAP: { [k: string]: string } = { b: "strong", i: "em" };
+const TAG_REGX: RegExp = /(<.{1,2}>)/;
```

Comment:
Nice, this is way better. We can modify this regex to explicitly capture <b>/<em>/{dice ...} later